specification_document-developments/2_0-Metabolomics-Draft/mzTab_format_specification_2_0-M_draft.adoc

```diff
@@ -1162,12 +1162,12 @@ If no identification has been performed at all then "no database" should be inse
 |*Mandatory* |True
 |*Example:* a|
 ....
-MTD database[1] [MIRIAM,MIR:00100079 , HMDB, ]
+MTD database[1] [MIRIAM, MIR:00100079, HMDB, ]
 MTD database[3] [, de novo , dn, ]
-MTD database[3] [MIRIAM, MIR:00000002 , CHEBI, ]
+MTD database[3] [MIRIAM, MIR:00000002, CHEBI, ]
 MTD database[4] [, mydatabase, mydb, ]
 OR
-MTD database[1] [, , "no database", null]
+MTD database[1] [, "no database", null, ]
```

Comment:
@oliveralka 
This needs to be a User Param with the name "no database" 
`[,, "no database", null]`
Otherwise, it would be a CV Param with an accession of "no database", which is illegal.

Reply:
Ah ok, will revert it- thanks!

Reply:
@nilshoffmann 
In the case of de novo and a custom database it is Ok? Since these are technically no CV Params:
`MTD database[2] [, de novo , dn, ]`
`MTD database[4] [, mydatabase, mydb, ]`

-> How would you specify the prefix then? 
`MTD database[2] [, , dn, ]`
`MTD database[4] [, , mydb, ]`

`MTD database[2]-prefix dn`
`MTD database[4]-prefix mydb`

Reply:
@oliveralka Your suggestions are fine! Please tell me, when your PRs are ready to be merged.